operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -569,6 +573,8 @@ protected V1ObjectMeta createMetadata() {
             .putLabelsItem(LabelConstants.DOMAINNAME_LABEL, getDomainName())
             .putLabelsItem(LabelConstants.SERVERNAME_LABEL, getServerName())
             .putLabelsItem(LabelConstants.CREATEDBYOPERATOR_LABEL, "true");
+    getPodLabels().forEach((k, v) -> metadata.putLabelsItem(k, v));
```

Comment:
Looking at this, I'm realizing that we need validation that the customer doesn't try to create a label matching one of the labels that we own.  The easiest rule would be that the customer cannot create a label starting with "weblogic."  Thoughts?